plugins/anonAuthPlugin.js

```diff
@@ -2,6 +2,7 @@ import Boom from 'boom';
 import chalk from 'chalk';
 import Rx from 'rxjs';
 
+const AUTH_TIMEOUT = 5000;
```

Comment:
yes - I'll put it [here](https://github.com/meetup/meetup-web-platform/blob/master/util/config.js)